cmd/erasure-healing-common.go

```diff
@@ -39,34 +38,21 @@ func commonTime(modTimes []time.Time, dataDirs []string) (modTime time.Time, dat
 		timeOccurenceMap[time.UnixNano()]++
 	}
 
-	for _, dataDir := range dataDirs {
-		if dataDir == errorDir {
-			continue
-		}
-		if dataDir == delMarkerDir {
-			dataDirOccurenceMap[delMarkerDir]++
-			continue
-		}
-		dataDirOccurenceMap[dataDir]++
-	}
-
 	// Find the common cardinality from previously collected
 	// occurrences of elements.
 	for nano, count := range timeOccurenceMap {
-		t := time.Unix(0, nano)
+		t := time.Unix(0, nano).UTC()
```

Comment:
Doesn't change anything functional but changed it to ensure that when we print for debugging both values that we match as well as this are in UTC() for ease to debugging.